### PR TITLE
sample:ipsp: remove undefined symbol and clarify how to build

### PR DESCRIPTION
**Pull request by jfischer-no**
Remove undefined symbol NET_DEBUG_L2_BT
Clarify how to build for Linux kernels released before 4.12.

### CODE REVIEW

File: samples/bluetooth/ipsp/README.rst [hunk 1]
```diff
@@ -14,6 +14,27 @@ Building and Running
...
+   :zephyr-app: samples/bluetooth/ipsp
+   :board: nrf52840_pca10056
+   :goals: build flash
+   :compact:
+
+Building and Running for Linux kernels released before 4.12
+===========================================================
+.. note::
+
+   For hosts using kernels released before 4.12 CONFIG_NET_L2_BT_ZEP1656
```
Comment (dbkinder) [inline samples/bluetooth/ipsp/README.rst]:
```suggestion
   For hosts using kernels released before 4.12,
   option :option:`CONFIG_NET_L2_BT_ZEP1656`
```

File: samples/bluetooth/ipsp/README.rst [hunk 2]
```diff
@@ -14,6 +14,27 @@ Building and Running
...
+   shall be selected: https://github.com/zephyrproject-rtos/zephyr/issues/3111
```
Comment (dbkinder) [inline samples/bluetooth/ipsp/README.rst]:
```suggestion
   must be selected.  For more information, see `Zephyr issue #3111
   <https://github.com/zephyrproject-rtos/zephyr/issues/3111>`_
```

**Review by dbkinder — CHANGES_REQUESTED** [review]

File: samples/bluetooth/ipsp/README.rst (hunk 2 above)
Comment (jfischer-phytec-iot) [reply samples/bluetooth/ipsp/README.rst]: done

**Review by dbkinder — APPROVED** [review]
doc changes LGTM, thanks!

**Review by nashif — APPROVED** [review]

**Pull request merged**